source/features/scrollable-code-and-blockquote.css

```
@@ -10,9 +9,6 @@
 	padding-bottom: 0.2em;
 }
 
-.comment-body details blockquote,
-.comment-body details pre,
-.comment-body blockquote blockquote,
-.comment-body blockquote pre {
+.comment-body :is(details blockquote, details pre, blockquote blockquote, blockquote pre) {
```

You need 2 is() here
```suggestion
.comment-body :is(details, blockquote) :is(blockquote, pre) {
```